document/web-api/index.bs

```diff
@@ -150,6 +152,8 @@ The [=serialization steps=], given |value|, |serialized|, and |forStorage|, are:
     1. Set |serialized|.\[[Bytes]] to the [=sub-serialization=] of |value|.\[[Bytes]].
 
     1. Set |serialized|.\[[AgentCluster]] to the [=current Realm=]'s corresponding [=agent cluster=].
+    
+    1. Set |serialized|.\[[Origin]] to be the [=origin-serialization=] of the user agent's [=origin=].
```

Comment:
A user agent doesn't have an origin - most likely this should be the [origin](https://html.spec.whatwg.org/multipage/webappapis.html#concept-settings-object-origin) of the [relevant settings object](https://html.spec.whatwg.org/multipage/webappapis.html#relevant-settings-object) for *value*.